torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -48,6 +48,105 @@ Value* broadcastSizes(at::ArrayRef<Value*> sizes, AliasDb* db) {
 }
 
 namespace tensorexpr {
+
+static const OperatorSet& supported_eltwise_set() {
```

Comment:
nit: `supported_eltwise_set` -> `supportedEltwiseSet` i think is more convention